s3/pkg/service/service.go

```diff
@@ -572,3 +571,16 @@ func HandleS3Error(err error, out *pb.BaseResponse) {
 		out.ErrorCode = int32(ErrorCodeResponse[ErrInternalError].HttpStatusCode)
 	}
 }
+
+func GetHttpErr(err error) (errCode int32, errMsg string) {
+	errCode = http.StatusInternalServerError
+	errMsg = ErrorCodeResponse[ErrInternalError].Description
+
+	s3err, ok := err.(S3ErrorCode)
+	if ok {
+		errCode = int32(ErrorCodeResponse[s3err].HttpStatusCode)
+		errMsg = ErrorCodeResponse[s3err].Description
```

Comment:
if message is being returned from the server, how will we handle a multi-lingual UI down the line ?

Reply:
Hi, no message returned now, see comment above please.